ipatests/test_integration/test_acme.py

```diff
@@ -252,3 +261,39 @@ def test_disable_acme_service(self):
             ['curl', '--fail', self.acme_server],
             ok_returncode=22,
         )
+
+
+class TestACMEwithExternalCA(TestACME):
+    """Test the FreeIPA ACME service with external CA"""
+
+    num_replicas = 0
+    num_clients = 1
+
+    @classmethod
+    def install(cls, mh):
+        # cache the acme service uri
+        acme_host = f'{IPA_CA_RECORD}.{cls.master.domain.name}'
+        cls.acme_server = f'https://{acme_host}/acme/directory'
+
+        # install packages before client install in case of IPA DNS problems
+        if not skip_certbot_tests:
+            cls.clients[0].run_command(['dnf', 'install', '-y', 'certbot'])
```

Comment:
Can you change the dnf calls to:

tasks.install_packages(cls.clients[0], ["certbot"])

I have a PR to this in the TestACME class.

Same w/mod_md below.